presto-docs/src/main/sphinx/functions/hive.rst

```diff
@@ -0,0 +1,46 @@
+==============================
+Hive Functions
+==============================
+
+Functions in Hive are categorized as either built-in or user-defined.
+Built-in functions are the ones already available in Hive and user-defined
+functions are custom functions for tasks like data cleansing and filtering.
+Hive UDFs can be defined according to programmer's requirements.
+
+Presto allows you to use the same UDFs that are available in your Hive deployment.
+All Hive UDFs may exist in Presto or external storage. Parameters and return values
+are converted between the types expected in Presto and Hive.
+
+Note: of the three supported Hive function types (scalar, windowing, and aggregate functions),
+only scalar functions are currently supported.
+
+Hive Scalar Functions
+---------------------
+
+.. note::
+
+    Hive has two different interfaces for writing Hive UDFs.
+    One is the basic UDF interface (deprecated) and the other
+    is the GenericUDF interface.
+
+    We currently only support the Hive built-in GenericUDF interface.
+
++---------------+-----------------------------------+-----------------------------------+
+|               | Hive Built-in Scalar Function     | External Hive Scalar Function     |
++---------------+-----------------------------------+-----------------------------------+
+| udf           | get_json_object                   | facebook_get_json_object          |
++---------------+-----------------------------------+-----------------------------------+
+| genericUdf    | str_to_map                        | facebook_str_to_map               |
++---------------+-----------------------------------+-----------------------------------+
+
+See the `Hive Language Manual <https://cwiki.apache.org/confluence/display/hive/languagemanual+udf>`_ for all hive scalar functions information.
+
+Examples
+--------
+
+Use hive function namespace ``hive.default`` to call the hive function.
```

Comment:
Is this plugin enabled by default or do people need to set it up? If people need to setup the Hive function namespace to use it, please also document how to enable it.

Reply:
I added the description of the Hive function namespace plugin.